scripts/build_utils.sh

```diff
@@ -423,7 +423,7 @@ delete_libvirt_vms() {
 
 clear_libvirt_networks() {
     # Sometimes, networks may linger around, so we must ensure they are killed:
-    networks=`sudo virsh net-list --all | grep active | egrep -v "(default|libvirt)" | cut -d ' ' -f 2`
+    networks=`sudo virsh net-list --all --name`
```

Comment:
what happens when you destroy the default one? all is good? I would've assumed that is needed